plugins/chruby/chruby.plugin.zsh

```diff
@@ -16,14 +16,15 @@ _source-from-omz-settings() {
 }
 
 _source-from-homebrew() {
-  (( $+commands[brew] )) || return 1
-
   local _brew_prefix
   # check default brew prefix
-  if [[ -h /usr/local/opt/chruby ]];then
-    _brew_prefix="/usr/local/opt/chruby"
+  if [[ -h /opt/homebrew/opt/chruby ]]; then
```

Comment:
This references the [new default Homebrew prefix](https://docs.brew.sh/Installation) on Apple Silicon, which is `/opt/homebrew`